svthalia/concrexit
# added regex for first/last name in registration form

Part of #3036 



### Summary
Added regex's to first and last name charfield to ensure no URLs are accepted as names and therefore not send to the Mail address as a clickable link. 

### How to test
Steps to test the changes you made:
1. Go to 'Become a Member'
2. Create a new member with the special characters (/,@,:,;,%,_,?,\,|) in the first and last name fields.
3. The registration should fail with an error message: "The first(last) name should not contain special characters like '/' or '@'."
